portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,27 +40,42 @@
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    String defaultCommentLimit = "5";
+    int commentLimit = Integer.parseInt(getParameter(request, "numShown", defaultCommentLimit));
```

Comment:
put orDefault in the method name or add comment /*default= */ defaultCommentLimit.

Reply:
Went with the former

Reply:
Went with the former